src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImage/index.tsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Button from 'app/components/button';
+import ClipboardTooltip from 'app/components/clipboardTooltip';
+import TextOverflow from 'app/components/textOverflow';
+import {IconStack} from 'app/icons';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Image} from 'app/types/debugImage';
+
+import layout from '../layout';
+import NotAvailable from '../notAvailable';
+import Stacktrace from '../stacktrace';
+import {combineStatus, getFileName} from '../utils';
+
+import Address from './address';
+import StacktraceStatusIcon from './stacktraceStatusIcon';
+import StatusTag from './statusTag';
+
+type Props = {
+  image: Image;
+  onOpenImageDetailsModal: (
+    image: Image,
+    imageAddress: React.ReactElement | null,
+    fileName?: string
+  ) => void;
+  style?: React.CSSProperties;
+};
+
+function DebugImage({image, onOpenImageDetailsModal, style}: Props) {
+  const {unwind_status, debug_status, code_file} = image;
+
+  const status = combineStatus(debug_status, unwind_status);
+
+  const fileName = getFileName(code_file);
+  const imageAddress = <Address image={image} />;
+
+  return (
+    <Wrapper style={style}>
+      <StatusColumn>
+        <StatusTag status={status} />
+      </StatusColumn>
+      <ImageColumn>
+        <ClipboardTooltip title={code_file} containerDisplayMode="inline-flex">
+          <FileName>{fileName}</FileName>
+        </ClipboardTooltip>
+        <ImageAddress>{imageAddress}</ImageAddress>
+      </ImageColumn>
+      <StacktraceColumn>
+        {debug_status || unwind_status ? (
+          <Stacktraces>
+            <Stacktrace
+              type="symbolication"
+              icon={<StacktraceStatusIcon status={debug_status} />}
+            />
+            <Stacktrace
+              type="stack-unwinding"
+              icon={<StacktraceStatusIcon status={unwind_status} />}
+            />
+          </Stacktraces>
+        ) : (
+          <NotAvailable />
+        )}
+      </StacktraceColumn>
+      <DebugFilesColumn>
+        <Button
+          size="xsmall"
+          icon={<IconStack size="xs" />}
+          onClick={() => onOpenImageDetailsModal(image, imageAddress, fileName)}
+        >
+          {t('View')}
+        </Button>
+        <Button
+          size="xsmall"
+          icon={<IconStack size="xs" />}
+          onClick={() => onOpenImageDetailsModal(image, imageAddress, fileName)}
+          label={t('View')}
+        />
+      </DebugFilesColumn>
+    </Wrapper>
+  );
+}
+
+export default DebugImage;
+
+const Wrapper = styled('div')`
+  :not(:last-child) {
+    > * {
+      border-bottom: 1px solid ${p => p.theme.border};
+    }
```

Comment:
```suggestion
      border-bottom: 1px solid ${p => p.theme.border};
```

To go with the above suggestion (if you choose to go with it).